.github/workflows/ci.yml

```diff
@@ -17,8 +17,7 @@ jobs:
 
     env:
       CI: true
-      # TODO: Rename the test docker image.
-      TEST_DOCKER_IMAGE: react-template
+      TEST_DOCKER_IMAGE: mat-process-thc
```

Comment:
I think we actually need to remove this file in favour of CircleCI config (see the top comment in this file).